src/index.html

```diff
@@ -4,10 +4,276 @@
+          <p class="text--price">$ 119.00</p>
+        </section>
+      </div>
+      <h2 class="title title--secondary" id="gallery">Browse Nothing products by category</h2>
+      <div class="gallery">
+        <div class="gallery__item gallery__item--big">
+          <img
+          srcset="./images/gallery-image-phone-small.jpg 320w, ./images/gallery-image-phone-large.jpg 1260w"
+          src="./images/gallery-image-phone-small.jpg"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          alt="Gallery photo phone"
+          class="gallery__image"
+          >
+        </div>
+        <div class="gallery__item gallery__item--small">
+          <img
+          srcset="./images/gallery-image-earphones-small.jpg 320w, ./images/gallery-image-earphones-large.jpg 1260w"
+          src="./images/gallery-image-earphones-small.jpg"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          alt="Gallery photo earphones"
+          class="gallery__image"
+          >
+        </div>
+        <p class="text--gallery">All products</p>
+      </div>
+      <div class="gallery">
+        <div class="gallery__item gallery__item--small">
+          <img
+          srcset="./images/gallery-image-earstick-single-small.jpg 320w, ./images/gallery-image-earstick-single-large.jpg 1260w"
+          src="./images/gallery-image-earstick-single-small.jpg"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          alt="Gallery photo single earstick"
+          class="gallery__image"
+          >
+        </div>
+        <div class="gallery__item gallery__item--big">
+          <img
+          srcset="./images/gallery-image-earstick-small.jpg 320w, ./images/gallery-image-earstick-large.jpg 1260w"
+          src="./images/gallery-image-earstick-small.jpg"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          alt="Gallery photo earsticks"
+          class="gallery__image"
+          >
+        </div>
+        <p class="text--gallery">Audio</p>
+      </div>
+      <div class="gallery">
+        <div class="gallery__item gallery__item--big">
+          <img
+          srcset="./images/gallery-image-phone2-small.jpg 320w, ./images/gallery-image-phone2-large.jpg 1260w"
+          src="./images/gallery-image-phone2-small.jpg"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          alt="Gallery photo phone"
+          class="gallery__image"
+          >
+        </div>
+        <div class="gallery__item gallery__item--small">
+          <img
+          srcset="./images/gallery-image-charger-small.jpg 320w, ./images/gallery-image-charger-large.jpg 1260w"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          src="./images/gallery-image-charger-large.jpg"
+          alt="Gallery photo charger"
+          class="gallery__image"
+          >
```

Comment:
please check formatting of your code
```suggestion
          <img
          srcset="./images/gallery-image-charger-small.jpg 320w, ./images/gallery-image-charger-large.jpg 1260w"
          sizes="(max-width: 1260px) 320px
                1260px"
          src="./images/gallery-image-charger-large.jpg"
          alt="Gallery photo charger"
          class="gallery__image"
          >
```